composer.json

```diff
@@ -25,10 +25,7 @@
     "autoload": {
         "psr-4": {
             "Spatie\\TwitterStreamingApi\\": "src"
-        },
-        "classmap": [
```

Comment:
This classmap is needed when the package is install into another project. 

Would love to know a better when of going about this, because this will break the CQ services in this package.